### PR TITLE
feat: added two new programs.

### PR DESCRIPTION
**Pull request by MarcosLudgerio**
Refs #8

### CODE REVIEW

File: src/assets/data/programs.json [hunk 1]
```diff
@@ -43,5 +43,23 @@
...
+    },
+    {
+        "title": "Web Summit",
+        "description": "Politico said we run “the world’s premier tech conference”, the Atlantic that Web Summit is “where the future goes to be born”, and Reuters that we assemble “the biggest names in tech and business”.",
+        "url": "https://websummit.com/rio/",
+        "end_date": "2023-05-01"
+    },
+    {
+        "title": "SÃO PAULO TECH WEEK",
+        "description": "Através de um evento totalmente dinâmico, a SPTW será 100% online e trará uma abordagem totalmente inovadora, onde cada membro poderá gerar conexões qualificadas para os seus negócios não apenas durante a semana de palestras, mas também ao longo do ano. ",
```
Comment (clebsonf) [inline src/assets/data/programs.json:55]: Amazing! But I will need you to put the descriptions in English.

**Review by clebsonf — CHANGES_REQUESTED** [review]
I will need you to put the descriptions in English.

File: src/assets/data/programs.json (hunk 1 above)
Comment (MarcosLudgerio) [reply src/assets/data/programs.json:55]: All rigth, thanks for your suggest. I will solve it!

**Comment by MarcosLudgerio** [comment]
@clebsonf can you review again, please?

**Review by clebsonf — APPROVED** [review]

**Pull request merged**